staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go

```diff
@@ -697,6 +699,9 @@ func (ss *scaleSet) listScaleSetVMs(scaleSetName, resourceGroup string) ([]compu
 	allVMs, rerr := ss.VirtualMachineScaleSetVMsClient.List(ctx, resourceGroup, scaleSetName, string(compute.InstanceView))
 	if rerr != nil {
 		klog.Errorf("VirtualMachineScaleSetVMsClient.List failed: %v", rerr)
+		if rerr.IsNotFound() {
+			return nil, ErrorVmssNotFound
```

Comment:
in this case, what about return `cloudprovider.InstanceNotFound` directly?

Reply:
updated accordingly

Reply:
thanks for the contribution